dti-website-redesign/src/components/Footer.tsx

```diff
@@ -1,3 +1,6 @@
+import { useState } from 'react';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
```

Comment:
Do we need to use a `Tooltip` component as opposed to any other div for the overlay? I think that component is used for text tooltips, and we are creating our own "tooltip" with the notification svg.